src/inngest/functions.ts

```
@@ -13,18 +14,35 @@ import { getSandBox, lastAssistantTexMessageContent, parseAgentOutput } from "./
 
+const endpoint =
+  "https://jk124-me7hp5up-swedencentral.cognitiveservices.azure.com/";
+const deployment = "gpt-4.1";
+const apiVersion = "2024-04-01-preview";
+
+// Function to sanitize content for PostgreSQL
+function sanitizeContent(content: string): string {
+  if (!content) return content;
+  // Remove null bytes and other problematic Unicode characters
+  return content
+    .replace(/\u0000/g, "") // Remove null bytes
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)